app/scripts/popup.js

```diff
@@ -4,12 +4,12 @@ const startPopup = require('./popup-core')
 const PortStream = require('./lib/port-stream.js')
 const isPopupOrNotification = require('./lib/is-popup-or-notification')
 const extension = require('extensionizer')
-const ExtensionPlatform = require('./platforms/extension')
+const WindowPlatform = require('./platforms/window')
 const NotificationManager = require('./lib/notification-manager')
 const notificationManager = new NotificationManager()
 
 // create platform global
-global.platform = new ExtensionPlatform()
+global.platform = new WindowPlatform()
```

Comment:
I would prefer not to make changes to the platform flag. This is rather important in case we want to differentiate functionality between `Window` and `Extension` in the future. 

Why does `openWindow` not work in ExtensionPlatform?